libraries/rust/src/lib.rs

```diff
@@ -1,19 +1,9 @@
-//! Welcome to pocketsizefund!
+///! Welcome to pocketsizefund!
```

Comment:
Why the third slash?

Reply:
`///!` is a special comment that denotes the entrypoint of the docs, basically it's like saying `this is the index.html`